Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1543,7 +1543,7 @@ def gen2018HiMix(fragment,howMuch):
         recoPremixUp15prod])
 
 recoPremixUp17prod = merge([
-        {'-s':'RAW2DIGI,L1Reco,RECO,EI,PAT'},
+        {'-s':'RAW2DIGI,L1Reco,RECO,RECOSIM,EI,PAT'},
         {'--datatier' : 'AODSIM,MINIAODSIM'},
         {'--eventcontent' : 'AODSIM,MINIAODSIM'},
         {'--era':'Run2_2017'},
```

Comment:
there are no modifications for the upgrade steps anymore.
Is this intended?
There was one in the previous iteration of this PR (as of e258549aee5bfe51bdf8204c569c240fcaba8648)

Reply:
It was intentional. I as said I remove all changes to the bare minimum that we need in the original requests. Until we have a solution that allows to have TPs in pdigi I see no reason to add RECOSIM everywhere. Now with hints from Matti we may have a solution. Until it's implemented I want this PR to stay clean and ready to be integrated AS-IS since it has everything that we were requesting. The rest is optional.

Reply:
no-pileup 2017 and classic mixing 2017 workflows are generated from the upgrade steps "RecoFull".
This is for most of the relval MC workflows now.
I think that we should have a RECOSIM in these workflows.